src/SFA.DAS.EAS.Portal.Worker/Program.cs

```diff
@@ -6,22 +6,40 @@
 using SFA.DAS.EAS.Portal.Worker.Startup;
 using SFA.DAS.EAS.Portal.Startup;
 using SFA.DAS.EAS.Portal.Worker.StartupJobs;
+using System;
 
 namespace SFA.DAS.EAS.Portal.Worker
 {
     class Program
     {
         static async Task Main(string[] args)
         {
-            using (var host = CreateHostBuilder(args).Build())
+            IHost host = null;
```

Comment:
Unhandled exceptions thrown by the console app are written to the console (including the stack trace), so these changes are unnecessary and introduce extra complexity.

Reply:
That wasn't happening locally.

Reply:
I did double check before making the suggestion 😃

Reply:
Removed the `TraceError`, as the exception will be logged to the set up nlog targets, currently console, log file and redis/kibana.